CMSIS/Core/Include/cmsis_iccarm.h

```diff
@@ -961,4 +961,6 @@ __packed struct  __iar_u32 { uint32_t v; };
 #pragma diag_default=Pe940
 #pragma diag_default=Pe177
 
+  #define __SXTB16_RORn __SXTB16(__ROR(op1, rotate))
```

Comment:
@TTornblom, can you please cross check that this macro definition works for IAR compiler?